config/samples/README.md

```diff
@@ -26,6 +26,13 @@ For more information, please visit the [gRPC OSS benchmarks README] on
 Examples for [PSM benchmarks](../../README.md#psm-benchmarks) can be found
 [here](templates/psm/README.md).
 
+[Prometheus](../prometheus/README.md) is enabled by default for all
```

Comment:
I would rewrite this a bit and point to a PR instead of a line of code.

```markdown
[Prometheus](../prometheus/README.md) is used to monitor CPU and memory
utilization in [PSM benchmarks](../../README.md#psm-benchmarks). Prometheus
monitoring can be enabled for any other test by adding the annotation
`enablePrometheus: 'true'` to the LoadTest configuration. For context,
see #315.
```